server/src/main/kotlin/com/studo/campusqr/database/AutomaticDataDeletion.kt

```diff
@@ -32,8 +32,8 @@ internal suspend fun automaticDataDeletion() {
     getCollection<SessionToken>().deleteMany(SessionToken::expiryDate lower now)
     getCollection<CheckIn>().deleteMany(CheckIn::date lower now.addDays(-deleteDays))
     getCollection<BackendAccess>().deleteMany(
-      BackendAccess::dateRanges.none(
-        DateRange::to greaterEquals now.addDays(-deleteDays)
+      BackendAccess::backendDateRanges.none(
```

Comment:
hmm I wouldn't rename the property. It's within BackendAccess, so it having the type List<BackendDateRange> is not surprising